core/Pos/Binary/Core/Genesis.hs

```diff
@@ -19,4 +19,7 @@ instance Bi GenesisWStakeholders where
 
 instance Bi GenesisDelegation where
     encode (unGenesisDelegation -> m) = encode (toList m)
-    decode = eitherToFail . mkGenesisDelegation =<< decode
+    decode =
+        -- dcNoCheck is not used here as genesis delegation is
+        -- constructed once only so these checks are cheap anyway.
```

Comment:
Doesn't matter, the checks should be disabled if I disable the checks.

Reply:
Btw, this module is unused and I deleted it in #1844.